crates/core_simd/src/masks.rs

```diff
@@ -88,6 +88,10 @@ impl_element! { isize }
 /// The layout of this type is unspecified, and may change between platforms
 /// and/or Rust versions, and code should not assume that it is equivalent to
 /// `[T; LANES]`.
+///
+/// For a type with layout guaranteed equivalent to `[T; LANES]`, use
+/// `Simd<T, LANES>`. For a type with layout guaranteed to use 1 bit per
+/// lane (padded up to full bytes), use `LANES::BitMask`.
```

Comment:
The type is actually `ToBitMask::BitMask`: https://doc.rust-lang.org/std/simd/trait.ToBitMask.html#associatedtype.BitMask.  It's not padded up to the next byte, but the next integer.